CHANGELOG.rst

```diff
@@ -1,6 +1,24 @@
 Changelog
 =========
 
+0.0.1.dev4 (2018-06-05)
+-----------------------
+
+Additions
+~~~~~~~~~
+
+* Merged gordon-janitor-gcp repo into gordon-gcp.
+* Added janitor plugin summaries.
+
+Changes
+~~~~~~~
+
+* Updated and fixed OWNERS.
+* Cleaned up some capitalizations and wordings.
+* Suppressed a test warning.
+* Fixed __all__s and exceptions.
```

Comment:
Could you add a tiny bit of context to this line? 

The _Cleaned up some capitalizations and wordings._ is super clear and informative, but this line has me wondering what does it even mean to fix an `__all__`.